src/solvers/flattening/boolbv_case.cpp

```diff
@@ -12,6 +12,8 @@ Author: Daniel Kroening, kroening@kroening.com
 
 bvt boolbvt::convert_case(const exprt &expr)
 {
+  PRECONDITION(expr.id() == ID_case);
```

Comment:
I don't know what that is, but can we introduce a `case_exprt` that has all these guarantees? Will require input from @kroening I suppose.

Reply:
`ID_case` does not seem to be assigned anywhere. @kroening: Can we remove `convert_case`/`ID_case`?